defaults/main.yml

```diff
@@ -1,21 +1,26 @@
 ---
-kea_apt_repo_version: 2-3
-kea_apt_key_fingerprint: 95B6C7ABDAEE9F164891B962DA05D46B7BABA24A
+kea_apt_repo_version: 2-4
+kea_apt_key_fingerprint: 4AD10B6C98E09742E530EC4F0D9D9A1439E23DB9
 kea_packages:
   - isc-kea-admin
   - isc-kea-hooks
 
+# which Kea components to enable
+kea_dhcp4_enabled: true
+kea_dhcp6_enabled: false
+kea_ddns_enabled: false
+kea_control_agent_enabled: false
+
 kea_dhcp4_package: >-
-  {{ 'isc-kea-dhcp4' if kea_apt_repo_version is version('2-3', '>=')
+  {{ 'isc-kea-dhcp4' if ansible_facts['distribution'] == 'Alpine'
   else 'isc-kea-dhcp4-server' }}
 kea_dhcp6_package: >-
-  {{ 'isc-kea-dhcp6' if kea_apt_repo_version is version('2-3', '>=')
+  {{ 'isc-kea-dhcp6' if ansible_facts['distribution'] == 'Alpine'
   else 'isc-kea-dhcp6-server' }}
 kea_ddns_package: >-
   {{ 'isc-kea-dhcp-ddns' if kea_apt_repo_version is version('2-3', '>=')
   else 'isc-kea-dhcp-ddns-server' }}
 kea_control_agent_package: isc-kea-ctrl-agent
-
```

Comment:
I think I would prefer to keep this whitespace to separate the variables into different sections in this file.